File: code/10/ruby/Duck.rb
# Duck.rb | Defines the Duck class which inherits attributes and methods
#   from the Bird superclass.
#
# Begun by: Dr. Nelesen, for CS 214 at Calvin College.
# Completed by: Mark Davis
# Date: April 21, 2016
####################################################

require './FlyingBird.rb'

class Duck < FlyingBird

def call
  'Quack'
end

end

File: code/10/java/Duck.java
/* Duck.java models a Duck
 *
 * Begun by: Dr. Nelesen, CS 214 at Calvin College.
 * Completed by: Mark Davis
 * Date: april 26, 2016
 */

public class Duck extends FlyingBird
{

//constructor for Duck
public Duck(String name) {
	super(name); //take over the BIRD constructor
}

//redefine call() for Duck
public String call() {
	return "Quack!";
}




}
